src/actions/__tests__/splitSentences.ts

```diff
@@ -647,3 +647,31 @@ describe('complicated cases', () => {
   - IPv4: 11.11.11.111`)
   })
 })
+
+describe('parenthetical content', () => {
```

Comment:
Tests look great and cover all relevant cases, positive and negative. Thank you!